src/py/aspen/config/production.py

```diff
@@ -1,8 +1,29 @@
-from .config import Config, DatabaseConfig
+import uuid
+from typing import Any, Mapping
+
+from .config import Config, DatabaseConfig, SecretsConfig
```

Comment:
should we change to aspen.config?

Reply:
ok you seem to really dislike relative imports, i will dump them all in a fire in a single PR coming up. :)

Reply:
no no no i was just thinking because now we have a smattering of relative imports and aspen imports. But a unifying PR _would_ be nice 😉

Reply:
believe it or not, there is actually a pattern and a rationale to when i prefer relative and absolute.